README.md

```diff
@@ -118,41 +126,76 @@ This uses a standard `rspec` and `rubocop` stack. To run these locally:
 
 ```bash
 bin/rake
+# or
+bundle exec rspec
+```
+
+### JavaScript
+
+```bash
+npm test
 ```
 
 ---
 
 ## Troubleshooting
 
-_I see Page Not Found when I log in and try to create a job listing_
+* I see Page Not Found when I log in and try to create a job listing.
 
-Try importing the school data if you have not already. When your sign in account was created, it was assigned to a
-school via a URN, and you may not have a school in your database with the same URN.
+Try importing the school data if you have not already. When your sign in account was created, it was assigned to a school via a URN, and you may not have a school in your database with the same URN.
 
 ---
 
-## Dependencies
+## Setup
+
+Welcome! :tada: :fireworks:
+
+As you are on-boarded, please make a note of any issues you come across, and update the [documentation](/documentation/new-joiners.md), to improve the process for the next person.
 
-### Baseline
+First, clone the project:
 
 ```bash
-Ruby 2.6.6
+git clone git@github.com:DFE-Digital/teacher-vacancy-service.git
+```
+
+If you are on a new device, remember to [generate a new SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+
+### Dependencies
+
+If you are on a Mac, you most likely have both PostgreSQL and rbenv (a Ruby version manager) installed already.
```

Comment:
I would standardise on using https://asdf-vm.com as it is multi language, we need ruby and nodejs